source/mir/random/engine/mersenne_twister.d

```diff
@@ -58,11 +58,12 @@ struct MersenneTwisterEngine(UIntType, size_t w, size_t n, size_t m, size_t r,
     /// The default seed value.
     enum UIntType defaultSeed = 5489;
 
-    private UIntType _z;
     /++
     Current reversed payload index with initial value equals to `n-1`
     +/
-    private UIntType index = void;
+    ptrdiff_t index = void;
```

Comment:
It is public to allow user to track Mt state. User may want to initialise it. Mir Random does not have tempalte range API, but we provide an ability for users to do what they want without copy-pasting code